compiler/plugins/target/AMD-AIE/iree-amd-aie/IR/AMDAIEOps.td

```diff
@@ -355,29 +355,60 @@ def AMDAIE_ChannelOp: AMDAIE_Op<"channel", [
 
     ```mlir
     %tile = amdaie.tile(%c0, %c0)
-    %channel = amdaie.channel(%tile, 0, port_type = DMA)
+    %channel = amdaie.channel(%tile, 0, port_type = DMA, direction = MM2S)
     ```
   }];
 
   let arguments = (
     ins Index:$tile,
         ConfinedAttr<I8Attr, [IntMinValue<0>]>:$value,
-        StrmSwPortTypeAttr:$port_type
+        StrmSwPortTypeAttr:$port_type,
+        DMAChannelDir:$direction
   );
 
   let extraClassDeclaration = [{
     TileOp getTileOp();
   }];
 
   let assemblyFormat = [{ 
-    `(` $tile `,` $value `,` `port_type` `=` $port_type `)` attr-dict
+    `(` 
+    $tile `,`
+    $value `,` 
+    `port_type` `=` $port_type `,` 
```

Comment:
Ideally this would be `strm_sw_port_type` to match `StrmSwPortType` in aie-rt? Is that too long you think?

Reply:
I think it's a bit long yeah, and also quite low level. Most people reading `strm_sw_port_type` will need to look up what it means anyway.